### ISSUE TITLE
The example in your zip does not work at all ...

### ISSUE THREAD

**Issue by tpk109** [xbgmsharp/videojs-watermark#9]

**Comment by xbgmsharp** [comment]
Please clarified what is not working.
I update to work the latset VideoJs player v5 and it is working fine.

**Comment by tpk109** [comment]
I just downloaded again and testet. In win10 and FF nothing happens when I
press the play button. I am using a Surface 3. Cause of the touchscreen it
behaves differently then PCs without touch, I experienced this with other
js-apps already.

I got it to work otherwise, but I can't remember, ages ago :-)

Perhaps it is worth mentioning in the docs that in iPhones and iPad,
WindowsPhone and probably Android the watermark does not make much sense,
as all these devices use their own player, some right from the start, some
when you switch to full screen.

This information would have saved me quite some time, we will now implement
the watermark with the server.

Thanks for your work anyhow.

2016-06-06 10:04 GMT+02:00 xbgmsharp <notifications@github.com>:

> Please clarified what is not working.
> I update to work the latset VideoJs player v5 and it is working fine.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbgmsharp/videojs-watermark/issues/9#issuecomment-223892899>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF1EvMSYB5SdUaZH9Z-flBjoQ6JVYc1Fks5qI9SfgaJpZM4H8yzY>
> .
>



-- 
------------------------------------------
Thomas Kratz

Webseiten, Programmierung, Grafik & (Online)-Marketing:
www.web-om.de

Niehoffs Blaike 5
48369 Saerbeck
02574 – 245873
0176- 9652 999 0
info@web-om.de
USt-IdNr: DE815141427
------------------------------------------
